src/App.tsx

```diff
@@ -1,24 +1,248 @@
-/* eslint-disable max-len */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, {
+  useEffect,
+  useMemo,
+  useRef,
+  useState,
+} from 'react';
 import { UserWarning } from './UserWarning';
+import { Todo } from './types/Todo';
+import { getpreparedTodos } from './utils/PreparedTodos';
+import { TodosFilter } from './types/TodoFilter';
+import {
+  addTodos,
+  deleteTodos,
+  getTodos,
+  updateTodo,
+} from './api/todos';
+import { TodoApp } from './components/TodoApp/TodoApp';
+import { TodoHeader } from './components/TodoHeader/TodoHeader';
+import { ErrorMessage } from './components/ErrorMessage/ErrorMessage';
+import { TempTodo } from './components/TempTodo/TempTodo';
+import { TodoFilter } from './components/TodoFilter/TodoFilter';
 
-const USER_ID = 0;
+const USER_ID = 11528;
 
 export const App: React.FC = () => {
+  const [todos, setTodo] = useState<Todo[]>([]);
+  const [filter, setFilter] = useState<TodosFilter>(TodosFilter.All);
+  const [errorMessage, setErrorMessage] = useState('');
+  const [tempTodo, setTempTodo] = useState<Todo | null>(null);
+  const [isLoading, setIsLoading] = useState(false);
+  const [request, setRequest] = useState(true);
+  const [loadingId, setLoadingId] = useState<number[]>([]);
+  const [isLoaderActive, setIsLoaderActive] = useState(false);
+  const [todoTitle, setTitle] = useState('');
+
+  const activeTodosCount = todos
+    .filter(({ completed }) => completed === false).length;
+
+  let completedTodosCount = todos
+    .some(({ completed }) => completed === true);
+
+  useEffect(() => {
+    getTodos(USER_ID)
+      .then(createdTodo => {
+        setTodo(createdTodo);
+        setRequest(false);
+      })
+      .catch(() => {
+        setErrorMessage('Unable to load todos');
+        setRequest(false);
+      });
+  }, []);
+
+  const idTimerRef = useRef<number>(0);
+
+  useEffect(() => {
+    if (idTimerRef.current) {
+      window.clearTimeout(idTimerRef.current);
+    }
+
+    idTimerRef.current = window.setTimeout(() => {
+      setErrorMessage('');
+    }, 3000);
+  }, [setErrorMessage]);
```

Comment:
https://github.com/mate-academy/react_todo-app-with-api/pull/801#discussion_r1337780293 - not fixed 😢 
Read the article at the link, it describes how to correctly work with timers in `useEffect`